https://bugs.gentoo.org/882787

a63b830554920476881837eeacd4a6b507632b19
Author:     Bruno Haible <bruno@clisp.org>
AuthorDate: Sun Aug 30 12:36:15 2020 +0200
Commit:     Bruno Haible <bruno@clisp.org>
CommitDate: Sun Aug 30 12:36:15 2020 +0200

Make the code C++17 compliant.

* lib/getline.cc (getstr): Don't use the 'register' keyword.
--- a/lib/getline.cc
+++ b/lib/getline.cc
@@ -55,7 +55,7 @@ getstr (char **lineptr, size_t *n, FILE *stream, char terminator, size_t offset)
 
   for (;;)
     {
-      register int c = getc (stream);
+      int c = getc (stream);
 
       /* We always want at least one char left in the buffer, since we
          always (unless we get an error while reading the first char)
